test/syscalls/linux/exec_binary.cc

```diff
@@ -438,7 +438,11 @@ TEST(ElfTest, MissingText) {
   ASSERT_THAT(RetryEINTR(waitpid)(child, &status, 0),
               SyscallSucceedsWithValue(child));
   // It runs off the end of the zeroes filling the end of the page.
+#if defined(__x86_64__)
   EXPECT_TRUE(WIFSIGNALED(status) && WTERMSIG(status) == SIGSEGV) << status;
+#elif defined(__aarch64__)
+  EXPECT_TRUE(WIFSIGNALED(status) && WTERMSIG(status) == SIGILL) << status;
```

Comment:
could you add a comment why we get SIGILL here? Is 0 an invalid instruction opcode on arm64?

Reply:
Yes. 0 is an invalid opcode.
Also, I have confirmed it in local with native mode.

Reply:
Done. Comment was added.
Thanks.